examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<blockquote>more work, and often requires you to pay for cloud storage, but it works on all kinds of TPU instances, and allows for truly massive data pipeline throughput.&nbsp;</blockquote><p> </p><p>Works on all kinds of devices (CPU, GPU, TPU). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Fixed!</p>